app/src/main/java/com/github/godspeed010/weblib/fragments/WebViewFragment.kt

```diff
@@ -88,21 +102,53 @@ class WebViewFragment : Fragment() {
 
                 Log.d(TAG, "URL CHANGE to $url")
 
-                if (view != null) {
-                    //Update address bar
-                    view.findViewById<EditText>(R.id.et_address_bar).setText(url)
+                //Update address bar
+                view.apply {
+                    findViewById<EditText>(R.id.et_address_bar).setText(url)
+                }
```

Comment:
`apply` is used to make code cleaner, easier to understand, and reduce the code you need to write. However, here it's doing the opposite.

Without an `apply`, the code would just be a single line: `view.findViewById<EditText>(R.id.et_address_bar).setText(url)`

However, now it has become three lines of code. If you had multiple properties you needed to change for `view`, then it would make sense to use `apply`, but here that is not the case.

```suggestion
                view.findViewById<EditText>(R.id.et_address_bar).setText(url)
```